ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -55,9 +57,16 @@ message CreatedEvent {
   Value contract_key = 7;
 
   // The arguments that have been used to create the contract.
-  // Required
+  // Set if there was a match by template_id in the given transaction filter.
+  // Optional
   Record create_arguments = 4;
```

Comment:
Shouldn't we populate this also for wildcard parties?
@meiersi-da @cocreature @stefanobaghino-da

Reply:
What do you mean by wildcard parties? Afaik, the ledger API does not support something like that.

Reply:
Oh you mean wildcard template ids, i.e., a party mapped to an empty filter? Yes, I think we should (and have to for backwards compat) populate it there.